pkgs/development/libraries/openexr/default.nix

```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
```

Comment:
I don't see why this is needed. Did you check, or added just in case?

Reply:
It is needed: the exr dylib name is set to its path in the temporary build directory, so we need this to change that to its path in the nix store.

Reply:
Sorry I got confused, it's the jxr that does that, the exr dylib name is set to just the name of the file, not the full path, so we need to change it to the full path in the nix store.

Reply:
I verified it by running `otool -l` on them.

Reply:
@uri-canva Sorry, I still don't get it. Here are my steps:

```
# nix-build -E 'with import ./nixpkgs {}; (openexr.overrideAttrs (old: { nativeBuildInputs = [ cmake ]; })).out' -o before/nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3
# nix-build -E 'with import ./nixpkgs {}; (openexr.overrideAttrs (old: { nativeBuildInputs = [ cmake fixDarwinDylibNames ]; })).out' -o after
/nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3
```

```
# diff -u <(otool -l before/lib/*.dylib) <(otool -l after/lib/*.dylib)
```

```diff
--- /dev/fd/13	2021-05-04 23:07:13.107059187 -0400
+++ /dev/fd/14	2021-05-04 23:07:13.107521240 -0400
@@ -1,4 +1,4 @@
-before/lib/libHalf-2_5.25.0.2.dylib:
+after/lib/libHalf-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -146,7 +146,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -239,8 +239,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libHalf-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libHalf-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -388,7 +388,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -481,8 +481,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libHalf-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libHalf-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -630,7 +630,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -723,8 +723,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libHalf.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libHalf.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -872,7 +872,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -965,8 +965,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIex-2_5.25.0.2.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIex-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -1150,7 +1150,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -1243,8 +1243,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIex-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIex-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -1428,7 +1428,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -1521,8 +1521,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIex-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIex-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -1706,7 +1706,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -1799,8 +1799,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIex.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIex.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -1984,7 +1984,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2077,8 +2077,8 @@
 Load command 16
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIexMath-2_5.25.0.2.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIexMath-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -2237,8 +2237,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2295,7 +2295,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2338,8 +2338,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIexMath-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIexMath-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -2498,8 +2498,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2556,7 +2556,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2599,8 +2599,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIexMath-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIexMath-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -2759,8 +2759,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2817,7 +2817,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -2860,8 +2860,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIexMath.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIexMath.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -3020,8 +3020,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3078,7 +3078,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3121,8 +3121,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImf-2_5.25.0.2.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImf-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -3329,8 +3329,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3387,14 +3387,14 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3408,21 +3408,21 @@
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3465,8 +3465,8 @@
 Load command 22
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImf-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImf-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -3673,8 +3673,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3731,14 +3731,14 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3752,21 +3752,21 @@
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -3809,8 +3809,8 @@
 Load command 22
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImf-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImf-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -4017,8 +4017,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4075,14 +4075,14 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4096,21 +4096,21 @@
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4153,8 +4153,8 @@
 Load command 22
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImf.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImf.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -4361,8 +4361,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4419,14 +4419,14 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4440,21 +4440,21 @@
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4497,8 +4497,8 @@
 Load command 22
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImfUtil-2_5.25.0.2.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImfUtil-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -4694,7 +4694,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 120
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4751,42 +4751,42 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -4836,8 +4836,8 @@
 Load command 23
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImfUtil-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImfUtil-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -5033,7 +5033,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 120
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -5090,42 +5090,42 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -5175,8 +5175,8 @@
 Load command 23
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImfUtil-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImfUtil-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -5372,7 +5372,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 120
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -5429,42 +5429,42 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -5514,8 +5514,8 @@
 Load command 23
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmImfUtil.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmImfUtil.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -5711,7 +5711,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 120
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImfUtil-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -5768,42 +5768,42 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmImf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 12
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 13
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 14
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -5853,8 +5853,8 @@
 Load command 23
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmThread-2_5.25.0.2.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmThread-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -6037,8 +6037,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6095,7 +6095,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6138,8 +6138,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmThread-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmThread-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -6322,8 +6322,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6380,7 +6380,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6423,8 +6423,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmThread-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmThread-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -6607,8 +6607,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6665,7 +6665,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6708,8 +6708,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libIlmThread.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libIlmThread.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 632
@@ -6892,8 +6892,8 @@
     flags 0x0
 Load command 3
           cmd LC_ID_DYLIB
-      cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIlmThread-2_5.25.dylib (offset 24)
+      cmdsize 120
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIlmThread-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6950,7 +6950,7 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -6993,8 +6993,8 @@
 Load command 17
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libImath-2_5.25.0.2.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libImath-2_5.25.0.2.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -7166,7 +7166,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -7223,21 +7223,21 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -7280,8 +7280,8 @@
 Load command 19
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libImath-2_5.25.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libImath-2_5.25.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -7453,7 +7453,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -7510,21 +7510,21 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -7567,8 +7567,8 @@
 Load command 19
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libImath-2_5.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libImath-2_5.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -7740,7 +7740,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -7797,21 +7797,21 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -7854,8 +7854,8 @@
 Load command 19
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
-before/lib/libImath.dylib:
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
+after/lib/libImath.dylib:
 Load command 0
       cmd LC_SEGMENT_64
   cmdsize 552
@@ -8027,7 +8027,7 @@
 Load command 3
           cmd LC_ID_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libImath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libImath-2_5.25.0.2.dylib (offset 24)
    time stamp 1 Wed Dec 31 19:00:01 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -8084,21 +8084,21 @@
 Load command 9
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libHalf-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 10
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIexMath-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
 Load command 11
           cmd LC_LOAD_DYLIB
       cmdsize 112
-         name /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
+         name /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib/libIex-2_5.25.dylib (offset 24)
    time stamp 2 Wed Dec 31 19:00:02 1969
       current version 25.0.2
 compatibility version 25.0.0
@@ -8141,4 +8141,4 @@
 Load command 19
           cmd LC_RPATH
       cmdsize 80
-         path /nix/store/9s1z0i90075adhhysmcz3k4mdx914bwq-openexr-2.5.3/lib (offset 12)
+         path /nix/store/8z07rx9pq7zvd50zlfwyml3i52jlda2k-openexr-2.5.3/lib (offset 12)
```

Reply:
You are right, I tested it again and it's not necessary, not sure why I added this sorry. Reverted.